src/index.js

```diff
@@ -0,0 +1,167 @@
+/* eslint-disable comma-dangle */
+/* eslint-disable no-continue */
+/* eslint-disable no-unused-vars */
+import './style.css';
+
+class Template {
+  constructor(description, completed, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+
+let array = [];
+const sendToLocalStorage = () => {
+  localStorage.setItem('list', JSON.stringify(array));
+};
+
+const section = document.querySelector('section');
+section.innerHTML = `
+  <div class="main-div">
+    <p id="header">Today's To Do <i class="fas fa-sync"></i></p>
+    <form class="form">
+      <input class="dataEntry" type="text" placeholder="Add to your list..." required></input>
+    </form>
+    <p id="clear">Clear all completed</p>
+  </div>
+`;
+
+// Create list
+const createList = () => {
+  const form = document.querySelector('.form');
+  const list = document.createElement('div');
+  list.className = 'input-div';
+  form.appendChild(list);
+  const checkboxes = document.createElement('input');
+  checkboxes.className = 'input';
+  checkboxes.type = 'checkbox';
+  const listText = document.createElement('p');
+  listText.className = 'listContent';
+  const threeDots = document.createElement('i');
+  threeDots.className = 'fas fa-ellipsis-v';
+  const trashIcon = document.createElement('i');
+  trashIcon.className = 'fas fa-trash-alt icon2';
+  list.append(checkboxes, listText, threeDots, trashIcon);
+
+  // Add event to checkboxes
+  // eslint-disable-next-line no-unused-vars
+  let count = 1;
+  checkboxes.addEventListener('click', () => {
+    threeDots.classList.toggle('remove-icon-active');
+    trashIcon.classList.toggle('icon2');
+    listText.classList.toggle('listContent-disable');
+    list.classList.toggle('changeBg');
+
+    const getting = JSON.parse(localStorage.getItem('list'));
+    const empty = [];
+    const hammasi = document.querySelectorAll('.input-div');
+
+    for (let i = 0; i < getting.length; i += 1) {
+      if (hammasi[i].classList.contains('changeBg')) {
+        getting[i].completed = true;
+        count += 1;
+      } else {
+        getting[i].completed = false;
+      }
+      empty.push(getting[i]);
+      localStorage.setItem('list', JSON.stringify(empty));
```

Comment:
Can we reuse the `sendToLocalStorage` function to avoid repetition? 
For example:
instead of `localStorage.setItem('list', JSON.stringify(empty));` call `sendToLocalStorage(empty)` function and modify the following:
```
const sendToLocalStorage = () => {
  localStorage.setItem('list', JSON.stringify(array));
};
``` 
to 
```
const sendToLocalStorage = (listStorage) => {
  localStorage.setItem('list', JSON.stringify(listStorage));
};
```